code/modules/mob/living/say.dm

```diff
@@ -225,8 +225,8 @@ GLOBAL_LIST_INIT(department_radio_keys, list(
 	for(var/_M in GLOB.player_list)
 		var/mob/M = _M
 		if(!M)
+			stack_trace("Null detected in player_list. Type: [GLOB.player_list[M]]")
 			listclearnulls(GLOB.player_list)

```

Comment:
You should probably remove this in case there's multiple nulls, which could be important info.